apps/cli/src/commands/bookmarks.ts

```diff
@@ -187,13 +187,19 @@ bookmarkCmd
   .description("update a bookmark")
   .option("--title <title>", "if set, the bookmark's title will be updated")
   .option("--note <note>", "if set, the bookmark's note will be updated")
+  .option("--stdin", "if set, the bookmark's note will be updated by stdin (not to be used with --note)")
   .option("--archive", "if set, the bookmark will be archived")
   .option("--no-archive", "if set, the bookmark will be unarchived")
   .option("--favourite", "if set, the bookmark will be favourited")
   .option("--no-favourite", "if set, the bookmark will be unfavourited")
   .argument("<id>", "the id of the bookmark to update")
   .action(async (id, opts) => {
     const api = getAPIClient();
+
+    if (opts.stdin) {
```

Comment:
Can we validate here that the `--note` flag is not set as well and print an error if it's not?

Reply:
Done